doc/api/stream.md

```diff
@@ -774,7 +774,7 @@ end
 *Note*: In general, the `readable.pipe()` and `'data'` event mechanisms are
 easier to understand than the `'readable'` event.
 However, handling `'readable'` might result in increased throughput.
-
+In the Meanwhile, new data will not trigger `'readable'` event everytime.
```

Comment:
* `meanwhile` rather than `Meanwhile`, although I'm not sure if `In the meanwhile` is necessary or clear. The word *meanwhile* suggests something else is happening. It's not clear *what* is happening. Maybe use "While" instead? Like "While the foo event is queued but has not yet been emitted, new data will not..."?

Smaller nits:

* ``trigger a `'readable'` event rather than ``trigger `'readable'` event`
* `every time` rather than `everytime`.
* blank line after this line

The nits can be handled by someone landing the PR but if you're going in to edit some more anyway, maybe take care of them then.